frontend/src/components/result-page/ResultPage.js

```diff
@@ -67,19 +68,20 @@ const ResultPage = ({ navigate }) => {
   return (
     <>
       <div>
-        <button className="results-page-home">Home</button>
+        <button className="results-page-home">
+          <img src={HomeIcon} alt="home" onClick={() => navigate("/")} className="home-icon" />
+        </button>
```

Comment:
this needs to be another component as we will probably use it in the display story so far page. it also needs to reset all the values in the local storage when pressed, so the api calls do not carry over leftover data from previous calls

Reply:
I've removed this now as Speaking to @natashabuckham she is going to create the home component for local storage capabilty too. So its a static icon